tests/phpunit/api/v3/CustomFieldTest.php

```diff
@@ -25,6 +26,14 @@ class api_v3_CustomFieldTest extends CiviUnitTestCase {
    * @throws \CRM_Core_Exception
    */
   public function tearDown(): void {
+    // The parent tearDown will delete the linked option group (gender)
+    // if we don't do it more carefully here.
+    CRM_Core_DAO::executeQuery('UPDATE civicrm_custom_field f
+      INNER JOIN civicrm_option_group g ON g.id = option_group_id
```

Comment:
What is this join for? Doesn't seem to be used?

Reply:
it's an inner join - aka a where

Reply:
Ok, I still think it's doing the same thing as `UPDATE civicrm_custom_field SET option_group_id = NULL` tho